Main/lib/perl/WorkflowSteps/MakeIprscan5NextflowConfig.pm

```diff
@@ -17,7 +17,7 @@ sub run {
     my $fastaSubsetSize = $self->getParamValue("fastaSubsetSize");
     my $appls = $self->getParamValue("appls");
     my $outputFile = $self->getParamValue("outputFile");
-    my $interproscanDatabase = $self->getParamValue("interproscanDatabase");
+    my $interproscanDatabase = join("/", $self->getSharedConfig("consign.pmacs.upenn.edu.softwareDatabasesDirectory"),$self->getSharedConfig("interproscanDatabaseDirectory"));
```

Comment:
consign.pmacs.upenn.edu should not be hard coded.  use the variable from the stepsShared.config

Reply:
This has been done.